README.md

```diff
@@ -462,32 +464,89 @@ values stored in `argv._`).  Set `desc` to `false` to create a hidden command.
 Hidden commands don't show up in the help output and aren't available for
 completion.
 
-Optionally, you can provide a handler `fn` which will be executed when
-a given command is provided. The handler will be called with `yargs` and
-`argv` as arguments.
-
-`yargs` is a blank instance of yargs, which can be used to compose a nested
-hierarchy of options handlers.
+Optionally, you can provide a `builder` object to give hints about the
+options that your command accepts:
 
-`argv` represents the arguments parsed prior to the
-command being executed (those described in the outer yargs instance).
+```js
+yargs.command('get', 'make a get HTTP request', {
+    url: {
+      default: 'http://yargs.js.org/'
+    }
+  })
+  .help()
+  .argv
+```
```

Comment:
I think it may be better to stick with the same structure of an example that we use above, i.e:

```js
var argv = require('yargs')
  .command('get', 'make a get HTTP request', {
    url: {
      default: 'http://yargs.js.org/'
    }
  })
  .help()
  .argv
```
The change in structure might confuse people (even though you mention that this is a `yargs` instance), and knowing people don't always read text and skip to code, we might run into some trouble :o What do you think?